Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/installing-boot9strap-(kartdlphax).txt

1. Wait for all checks to complete
1. When prompted, input the key combo given on the top screen to install boot9strap

> 1. Once it has been completed, press (A) to reboot your device

Reviewer: ```suggestion
1. Once it is complete, press (A) to reboot your device
```
